File: java/src/MainBatallaCarnotronGallotronix.java

import model.batalla.Batalla;
import model.chimpos.Carnotron;
import model.chimpos.Gallotronix;
import model.logger.Logger;
import model.logger.Nivel;
import model.referi.Referi;
import model.tipo.TipoChinpokomon;

public class MainBatallaCarnotronGallotronix {

    public static void main(String[] args) {
        Logger.getInstance(Nivel.INFO);
        Gallotronix gallotronix = new Gallotronix("Gallotronix", 5, TipoChinpokomon.ROBOT);
        Carnotron carnotron = new Carnotron("Carnotron", 6, TipoChinpokomon.ANIMAL);
        Batalla batalla = new Batalla(carnotron, gallotronix);
        Referi referi = new Referi(batalla);
        referi.gestionarBatalla();
    }

}

File: python/MainBatallaCarnotronGallotronix.py
from model.logger.logger import Logger
from model.logger.nivel import Nivel
from model.chipos.Carnotron import Carnotron
from model.chipos.Gallotronix import Gallotronix
from model.chipos.enums import TiposDeChipos
from model.batalla.batalla import Batalla
from model.referi.referi import Referi


logger = Logger(Nivel.INFO)
gallotronix = Gallotronix("Gallotronix", 5, TiposDeChipos.ROBOT)
carnotron = Carnotron("Carnotron", 6, TiposDeChipos.ANIMAL)
batalla = Batalla(carnotron, gallotronix)
referi = Referi(batalla)
referi.gestionar_batalla()
